scheduler/src/cook/mesos/api.clj

```diff
@@ -872,7 +872,7 @@
                         jobs (get params :jobs)
                         groups (get params :groups)
                         user (get-in ctx [:request :authorization/user])
-                        override-group-immutability? (boolean (get params override-group-str))]
+                        override-group-immutability? (boolean (get params :override-group-immutability))]
```

Comment:
Are we breaking anything by expecting a keyword instead of a string in params?

Unit tests for the change would be consoling.